### DISCUSSION TITLE
Crazyflie client shutdown every time I try to connect the crazyflie (emergency stop)

### DISCUSSION THREAD

**Discussion by JaouaherBelgacem** [bitcraze/discussions#833] [Q&A]
Hey everyone,
I am using crazyflie for location tracking project and it was working quite good in April and once I got back to work on it last week, it’s refusing to connect. Every time I press connect the crazyflie client shutdown and it indicates an emergency stop. I tied to upgrade the firmware one, and reinstalled everything but it’s not working any help please! 
Thanks in advance

**Comment by knmcguire** [comment]
I see you've asked the same question here: https://github.com/orgs/bitcraze/discussions/831 so let's communicate further there
